internal/core/fixtures/cluster-get-west.json

```
@@ -11,11 +11,16 @@
             "name": "capacity",
             "unit": "B",
             "capacity": 185,
-            "comment": "hand-counted",
             "domains_quota": 50,
             "usage": 8,
             "physical_usage": 7
           },
+          {
+            "name": "capacity_portion",
+            "unit": "B",
+            "contained_in": "capacity_portion",
```

A resource cannot be contained in itself:
```suggestion
            "contained_in": "capacity",
```
There are multiple instances of this across all the fixture files. I don't think it affects the test results, though, since `contained_in` is not evaluated by limesctl yet.

---

The fixtures were copied directly from the Limes repository. I've created a PR to also make the corresponding changes in Limes.